services/events-service/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.19-alpine as app-builder
+WORKDIR /go/src/app
+COPY . .
+RUN apk add git
+# Static build required so that we can safely copy the binary over.
+# `-tags timetzdata` embeds zone info from the "time/tzdata" package.
+RUN CGO_ENABLED=0 go install -ldflags '-extldflags "-static"' -tags timetzdata
+
+FROM scratch
+# the test program:
+COPY --from=app-builder /go/bin/events-service /events-service
+# the tls certificates:
+# NB: this pulls directly from the upstream image, which already has ca-certificates:
+COPY --from=alpine:latest /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+ENTRYPOINT ["/events-service"]
```

Comment:
Can you add a new line to the end of this file? Otherwise all good 😎

Reply:
Bro you're too fast but yup 😎

Reply:
Alright updated 😎